src/features/game/state/GameStateManager.ts

```diff
@@ -82,6 +84,8 @@ class GameStateManager {
         this.checkpointTask.showTask(task);
       });
 
+    this.quizScores = new Map<string, number>(this.getSaveManager().getQuizScores());
```

Comment:
Unnecessary type argument as the typing can be inferred.

Reply:
Comment addressed: commit cacd2f2d98f27ef944ef885fbd582464269edd31